inline-caching/cache-loader/src/main/java/io/vmware/event/ItemAsyncEventListener.java

```diff
@@ -0,0 +1,84 @@
+package io.vmware.event;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.Record;
+import org.jooq.SQLDialect;
+import org.jooq.Table;
+import org.jooq.impl.DSL;
+
+import org.apache.geode.LogWriter;
+import org.apache.geode.cache.CacheFactory;
+import org.apache.geode.cache.Operation;
+import org.apache.geode.cache.asyncqueue.AsyncEvent;
+import org.apache.geode.cache.asyncqueue.AsyncEventListener;
+
+
+@SuppressWarnings("deprecation")
+public class ItemAsyncEventListener implements AsyncEventListener {
+
+	private static LogWriter log;
+	
+	static {
+		log = CacheFactory.getAnyInstance().getDistributedSystem().getLogWriter();
+	}
+
+	@Override
+	public boolean processEvents(List<AsyncEvent> events) {
+		try(Connection conn = ConnectionPool.getInstance().getConnection()) {
+			log.info("connected:"+ conn.getSchema());
+			return process(conn, events);
+		} catch (SQLException e) {
+			log.error("Exception", e);
+			return false;
+		}
+	}
+
+	private boolean process(Connection conn, List<AsyncEvent> events) {
+		DSLContext create = DSL.using(conn, SQLDialect.POSTGRES);
```

Comment:
`create` does not represent anything useful, it is a verb and not a noun

Reply:
renamed